api/stories/newStoryRouter.js

```diff
@@ -10,7 +10,7 @@ router.get('/:id', (req, res) => {
 });
```

Comment:
You need to run the lint script to address these formatting errors.

Reply:
Thanks for your feedback. Will do so asap

Reply:
Yes that will be a quick fix.